packages/cardhost/app/components/card-manipulator.js

```diff
@@ -102,6 +109,8 @@ export default class CardManipulator extends Component {
   @tracked card;
   @tracked selectedField;
   @tracked isDragging;
+  @tracked selectedFieldName;
+  @tracked selectedFieldLabel;
```

Comment:
We shouldn't have to track these two prperties, as they are already being tracked in the @selectedField's `Field` class

Reply:
like the field's name in the right edge was updating in tests in master when you click on the field in the card, as well as the type and needed when embedded. that all happens because we are already tracking these properties in the `Field` class.

Reply:
I think the problem is actually in the FieldRenderer--we are not treating the field label's input the same as the field name's input when doing one-way binding

Reply:
Most likely we should add a {{did-update}} render modifier for dealing with this...